tests/test_auth_cli.py

```diff
@@ -7,6 +7,15 @@
 from ggshield.config import Config
 
 
+_TOKEN_RESPONSE_PAYLOAD = {
+    "type": "personal_access_token",
+    "account_id": 17,
+    "name": "key",
+    "scope": ["scan"],
+    "expire_at": None,
+}
+
+
 @pytest.fixture(autouse=True)
 def tmp_config(monkeypatch, tmp_path):
     monkeypatch.setattr("ggshield.config.get_auth_config_dir", lambda: str(tmp_path))
```

Comment:
The `TestAuthLoginToken` class was supposed to be restricted to the `--method=token` flow. You should either rename it or create a `TestAuthLoginWeb` class.